src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2629,6 +2630,29 @@ public static class PKAnalysis
          public static readonly string Unit = UI.Unit;
          public static readonly string Description = "Description";
          public static readonly string Warning = "Warning";
+         public static readonly string OnCurve = "On curves";
+         public static readonly string OnIndividuals = "On individuals";
+         public static string LowerSuffix(string text)
+         {
+            var regex = new Regex(@"^(.*)Range (\d*)% to (\d*)%");
+            var match = regex.Match(text);
+
+            if (!match.Success)
+               return text;
+
+            return $"{match.Groups[1]}{match.Groups[2]}%";
+         }
+
+         public static string UpperSuffix(string text)
+         {
+            var regex = new Regex(@"(.*)Range (\d*)% to (\d*)%");
```

Comment:
This file should really be only about constant string and putting strings together. Think almost this is the english version of PKSim. The code with regex etc.. does not belong here.
Note on the regex .They are almost identical but not quite. There is not comment explaining what it does and the code is almost exactly the same. I think this needs to be refactored. 
And this needs to be commented, if we really need regex for this